src/core-components/Alert/Alert.less

```diff
@@ -0,0 +1,14 @@
+@import '~antd/dist/antd.less';
+
+.ant-alert {
+  border-radius: 8px !important;
+  color: @alert-message-color !important;
```

Comment:
You redefining `@alert-message-color` in `craco.config` and then using the default `antd` color in local component's styles.
What's the reason?

Reply:
@alert-message-color in local component's styles represents the color defined in craco.config. I checked it.
And it is used in .ant-alert class in Alert.less in order to define the color of description text.